learn/security/managing_api_keys.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Managing API keys — Meilisearch documentation
+description: This guide shows you how to access and configure API keys.
+---
+
+# Managing API keys
+
+This guide shows you how to access and configure API keys.
+
+## Accessing the `/keys` API route
+
+You can access the [`/keys` route](/reference/api/keys) using the master key or an API key with access to the `keys.get`, `keys.create`, `keys.update`, or `keys.delete` actions.
+
+## Updating an API key
+
+You can only update the `name` and `description` of an API key, even after it expires.
+
+For example, we can update the `Default Search API Key` and change its description:
+
+<CodeSamples id="security_guide_update_key_1" />
+
+```json
+{
+  "name": "Default Search API Key",
+  "description": "Default Search API Key",
+  "key": "d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4",
+  "uid":"74c9c733-3368-4738-bbe5-1d18a5fecb37",
+  "actions": [
+    "search"
+  ],
+  "indexes": [
+    "*"
+  ],
+  "expiresAt": null,
+  "createdAt": "2022-01-01T10:00:00Z",
+  "updatedAt": "2022-01-01T10:00:00Z"
+}
+```
+
+To update an API key, you must use the [update API key endpoint](/reference/api/keys#update-a-key), which can only be accessed with the master key or an API key with the `keys.update` action.
+
+Meilisearch supports partial updates with the `PATCH` route. This means your payload only needs to contain the data you want to update—in this case, `description`.
+
+## Creating an API key
```

Comment:
Is there a reason for putting the Update section before the Creation section?

Reply:
I also find the order a little bit disturbing, I'd start with listing, then creating, updating and deleting

Reply:
I'm not sure what was the reasoning behind the order here. It might have been because we already start with two API keys, so creation is not necessarily the first thing a user will do? In any case, I was trying to avoid opening the can of worms that is this page (my controversial take is we need to delete this page and write a new guide).

Since we have two votes for changing the order, though, I'll follow the order suggested by @CaroFG. It's also the order we have in the reference.